lib/state/app_state.dart

```diff
@@ -8,7 +8,7 @@ part 'app_state.g.dart';
 class AppState with _$AppState {
   factory AppState({
     /// TODO: this is for testing purposes and will be changed once the state is finalized
-    @Default('') @JsonKey(ignore: true) String name,
+    @Default('') @JsonKey(ignore: true) String pokemons,
```

Comment:
just revert everything to its previous state => `@Default('') @JsonKey(ignore: true) String name,` you don't need this for the simulation you made for go_router setup right? 🙂